Build/MQTTnet.nuspec

```diff
@@ -22,6 +22,7 @@
 * [Server] Added interceptor for unsubscriptions.
 * [MQTTnet.Server] Added interceptor for unsubscriptions.
 * [MQTTnet.AspNetCore] improved compatibility with AspNetCore 3.1
+* [MqttApplicationMessage] Added GetUserProperty<T> convenience method (thanks to @PMExtra).
```

Comment:
Please choose either Core, Client, Server, or a project name like MQTTnet.Server. In your case I would say "Core" fits best.